packages/smooth_app/lib/pages/product_page.dart

```diff
@@ -374,6 +377,34 @@ class _ProductPageState extends State<ProductPage> {
       listItems.add(_getAttributeGroupWidget(attributeGroup, iconWidth));
     }
 
+    if (_product.categoriesTags != null) {
+      for (int i = 0; i < _product.categoriesTags.length; i++) {
```

Comment:
We can leave this as is for now, but the categories at the very top probably are not very useful to show.